src/client/utils/dom.ts

```diff
@@ -819,9 +819,7 @@ export function isInputWithoutSelectionProperties (el): boolean {
     if (!isNumberOrEmailInput(el))
         return false;
 
-    const hasSelectionProperties = typeof el.selectionStart === 'number' &&
-        typeof el.selectionEnd === 'number' &&
-        typeof el.selectionDirection === 'string';
+    const hasSelectionProperties = typeof el.selectionStart === 'number' && typeof el.selectionEnd === 'number';
```

Comment:
Add a client test for the `isInputWithoutSelectionProperties` function into https://github.com/DevExpress/testcafe-hammerhead/blob/master/test/client/fixtures/utils/dom-test.js file.